From e82b96721d3537c68b6197647f6eaa8a103d2289 Mon Sep 17 00:00:00 2001
From: Loc Pham <loc.pham.xh@renesas.com>
Date: Wed, 10 Jul 2019 13:08:04 +0700
Subject: [PATCH] connmand: systemd: Do not act as a DNS proxy

By default, ConnMan will direct all DNS traffic to itself by setting nameserver to 127.0.0.1 in resolv.conf file.
This feature should be disabled. ConnMan should use remote DNS servers, such as: 8.8.8.8 and 8.8.4.4 instead because
Docker needs these DNS servers to pull images.

Signed-off-by: Loc Pham <loc.pham.xh@renesas.com>
---
 src/connman.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index dab48bc..50437e4 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -12,7 +12,7 @@ Conflicts=systemd-resolved.service
 Type=dbus
 BusName=net.connman
 Restart=on-failure
-ExecStart=@sbindir@/connmand -n
+ExecStart=@sbindir@/connmand -n --nodnsproxy
 StandardOutput=null
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE CAP_NET_RAW CAP_SYS_TIME CAP_SYS_MODULE
 ProtectHome=true
-- 
2.7.4

